challenges/test/lib/src/test/java/code/treetest/TreeTest.java

```diff
@@ -134,5 +134,39 @@ public void binarySearchTreeTest(){
 
     assertTrue(searchTree.contains(7));
     assertFalse(searchTree.contains(-4));
+    System.out.println(searchTree.breadthFirst());
+  }
+
+  @Test public void treeMaxinumValueTest(){
```

Comment:
Good work with the test. I'm curious what happens if two values are the same for the maxval?